returnn/config.py

```diff
@@ -190,7 +190,11 @@ def add_line(self, key, value):
       if value_type == str:
         pass  # keep as-is
       else:
-        value = eval(value)
+        try:
+          value = eval(value)
+        except:
+          from returnn.log import log
+          print("WARNING: can't evaluate config param '{}' to previous type: {}. Keeping as string.".format(value, value_type),file=log.v1)
```

Comment:
You should keep the code consistent to our other code. We use the `%` formatting for strings.

Also, you should follow PEP8. There is a missing space. But PyCharm also should tell you about this. See our contribution rules.